infrastructure/local/peer-docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       dockerfile: ./packages/auth/Dockerfile
     restart: always
     networks:
-      rafiki:
+      local_rafiki:
```

Comment:
Nit pick: I'm always confused when I see the colon at the end of the network name. Could we not do the following?
```
networks:
  - local_rafiki
```